Dockerfile

```diff
@@ -20,8 +20,11 @@ COPY . .
 # CGO_ENABLED and installsuffix are part of the scheme to get better caching on builds
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LDFLAGS}" -a -installsuffix cgo -o /go/bin/zqd ./ppl/cmd/zqd
 
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LDFLAGS}" -a -installsuffix cgo -o /go/bin/pgctl ./ppl/cmd/pgctl
 
 FROM scratch
 WORKDIR /app
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /go/bin/zqd /app
+COPY --from=build /go/bin/pgctl /app
+COPY --from=build /build/ppl/zqd/db/postgresdb/migrations /postgres/migrations
```

Comment:
These changes to the Docker build are OK, but in the future we should not include the migration information except for in the containers that require it. So, for example, the containers for the workers and recruiter which do not access postgres would not have the migration code included. Doing that is more complicated, so I think this code is best for now.

Reply:
Agreed.